brave/browser-laptop
# Refine tab's element position

Auditors: @bsclifton, @bbondy, @bradleyrichter, @jkup

Fix #5431
Fix #6511

This PR follows this @bradleyrichter spec for tabs:

![brad-tab-spec](https://cloud.githubusercontent.com/assets/4672033/21078464/3bc3f0c0-bf57-11e6-8d39-e2c868040ea8.png)

**Test Plan:**

* All previous tab's test must pass:

`npm run test -- --grep="tab tests"`

**Manual Tests**

* Open 20 tabs with the following included:
* new session with audio
* new session without audio
* private with audio
* private without audio
* default with audio
* default no audio
* about:newtab or about:blank
* Add any other 2 tabs
* Resize window's width
* At each random breakpoint, tab's content should work based on above spec
* Icons must be hidden/shown gracefully (no icons overflowing tab's container)
* There should be no UI inconsistency
